server/public/styles.css

```diff
@@ -0,0 +1,37 @@
+*{
+  margin: 0;
+  box-sizing: border-box;
+}
+header{
+  background-color: rgb(46,51,56);
+  color: white;
+  padding: 0.5rem;
+}
+header>p{
+  margin:5px;
+  padding-left:2rem;
+}
+i{
+  margin:5px;
+}
+body{
+  background-color: rgb(247,248,249);
+}
+.container{
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 2rem;
+}
+.card.col{
+flex-basis: 30%;
+margin: 1rem;
+}
```

Comment:
Be sure not to style Bootstrap's classes yourself. You can probably achieve these with flex and spacing helpers.

https://getbootstrap.com/docs/4.4/utilities/flex/
https://getbootstrap.com/docs/4.4/utilities/spacing/